mkdocs.yml

```diff
@@ -193,3 +193,4 @@ nav:
       - developers/brand-guide.md
       - developers/testing-docs.md
       - developers/writing-style-guide.md
+      - developers/remote-config.md
```

Comment:
Why is this sneaking in here?

Reply:
I snuck it in there due to watching `make mkdocs-serve` and its complaint. Hand slap.